docs/runtimes/spark-operator.ipynb

```diff
@@ -79,7 +79,9 @@
     "# command will use our spark code which needs to be located on our file system\n",
     "# the name param can have only non capital letters (k8s convention)\n",
     "read_csv_filepath = os.path.join(os.path.abspath(\".\"), \"spark_read_csv.py\")\n",
-    "sj = mlrun.new_function(kind=\"spark\", command=read_csv_filepath, name=\"sparkreadcsv\")\n",
+    "sj = mlrun.project.set_function(\n",
```

Comment:
Also here, pretty sure you need to have a project instance first and not do `mlrun.project`